assessment.md

```diff
@@ -119,56 +119,57 @@ when creating python programs.
-  * A = Pushed to correct repository with descriptive commits
+  * N = GitHub flow model unused or no commits to the Github repository
+  * I = Pushed to master or non-descriptive commits to the Github repository
+  * A = Pushed to correct repository with descriptive commits to the Github
+  repository
   * G = Use of a branch/fork demonstrated appropriately
   (e.g having a feature branch) along with descriptive commits
   * E = Use of multiple useful branches/forks demonstrated appropriately
-    containing multiple coherent and descriptive commits
+    containing multiple consistent and descriptive commits
 * Creating and using a repository branch
   * N = Did not create or utilize branches
-  * I = Create unclear branches
-  * A = Create too many poorly named branches
-  * G = Create useful adequately named branches
-  * E = Create useful and descriptive branches
+  * I = Created unclear branches
+  * A = Created too many poorly named branches
+  * G = Created useful adequately named branches
+  * E = Created useful and descriptive branches
 * Creating and using a repository fork
   * N = Did not create or utilize forks
-  * I = Create unclear forks
-  * A = Create too many poorly named forks
-  * G = Create useful adequately named forks
-  * E = Create useful and descriptive forks
+  * I = Created unclear forks
+  * A = Created too many poorly named forks
+  * G = Created useful adequately named forks
+  * E = Created useful and descriptive forks
 * Merging a branch or a fork to another branch or fork
   * N = Not communicating changes before merging branches and forks
-  * I = Merging with minimal communication or merging faulty code
+  * I = Merging with minimal communication or merging defective code
   * A = Communicating changes with the team and reconciling differences
   * G = Merging branches after extensive communication
   * E = Merging branches with well-documented, detailed code and
-    extensive communication
+    extensive team communication
 * Creating and reviewing a pull request
-  * N = No PR made
+  * N = No PR(Pull Request) made
   * I = Does not tag the related issues it is trying to close with poor description
   * A = Tags any related issues it is trying to close, describes
     proposed changes by a person
   * G = Tags any related issues it is trying to close, describes most proposed changes,
-    discusses results of PR with Travis CI
+    and passes Travis CI
   * E = Tags any related issues it is trying to close, fully describes all
     proposed changes and passes Travis CI
 * Using appropriate commit messages
-  * N = Commit messages are nonsensical
+  * N = Commit messages do not make sense
   * I = Commit messages are uninformative
   * A = Commit messages are sensible but not descriptive
-  * G = Commit messages are both clear, and moderately descriptive
-  * E = Commit messages are clear, concise and descriptive
+  * G = Commit messages are clear, concise and moderately descriptive
```

Comment:
The addition of concise here, in my opinion, makes the difference between level `G` and `E` too small. I think the original wording is better. This is of course the choice of the students involved, however.